src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -1046,6 +1090,19 @@ private bool ShouldExecuteSyntaxNodeActions(AnalysisScope analysisScope)
             return false;
         }
 
+        private bool ShouldExecuteOperationActions(AnalysisScope analysisScope)
+        {
+            foreach (var analyzer in analysisScope.Analyzers)
+            {
+                if (this.OperationActionsByAnalyzerAndKind.ContainsKey(analyzer))
+                {
+                    return true;
+                }
+            }
+
+            return false;
```

Comment:
I think the LINQ form forces the creation of a closure for the lambda. Where do we usually land with respect to concise expression vs. hidden cost?